tests/platform_tests/test_intf_fec.py

```diff
@@ -19,6 +19,15 @@
 ]
 
 
+def is_supported_platform(duthost):
+    if any(platform in duthost.facts['platform'] for platform in SUPPORTED_PLATFORMS):
+        skip_release(duthost, ["201811", "201911", "202012", "202205", "202211", "202305"])
+        return True
+    else:
+        pytest.skip("DUT has platform {}, test is not supported".format(duthost.facts['platform']))
+        return False
```

Comment:
no need to return false if it's already skipped, right?

Reply:
Fixed it. Thanks!